src/main/java/orangelynx/ic2_extra_fuels/Ic2ExtraFuels.java

```diff
@@ -59,9 +61,11 @@ public void preInit(FMLPreInitializationEvent event)
 
             String[] nextLine;
             while ((nextLine = additionalFuelsConfigFileReader.readNext()) != null) {
+                int heatPerTick = nextLine.length > 3 ? Integer.parseInt(nextLine[3]) : 0;
```

Comment:
I appreciate the thought of backward compatibility / flexibility here, but I'd rather keep it consistent. If the csv format allows for heatPerTick in an extra (4th) column, it should be populated in all lines, since AFAIK incomplete rows are not a specified / common feature of csv. Also I think we can afford to loose backward compatibility at this point. 

```suggestion
```